config/sample_manifest_excel/manifest_types.yml

```diff
@@ -679,7 +679,6 @@ heron:
     - :subject
     - :disease
     - :sample_ebi_accession_number
-    - :donor_id2
```

Comment:
Why was this not appearing in the Heron manifest?

Reply:
Because the column configuration contained a name: attribute, which meant it was over-writing the donor_id column definition instead. It appears that if a column definition is missing, we just skip the column. (Didn't look into exactly what happens here too closely. )